src/js/utils/events.js

```diff
@@ -198,29 +201,35 @@ export function fixEvent(event) {
     }
   }
 
+  event.fixed_ = true;
   // Returns fixed-up instance
   return event;
 }
 
 /**
  * Whether passive event listeners are supported
  */
-let _supportsPassive = false;
-
-(function() {
-  try {
-    const opts = Object.defineProperty({}, 'passive', {
-      get() {
-        _supportsPassive = true;
-      }
-    });
+let _supportsPassive;
+
+const supportsPassive = function() {
```

Comment:
For some reason this takes about 2ms to complete, making it lazy makes it take much less time later when an event that would need to be passive is actual used.